README.md

```diff
@@ -126,7 +126,18 @@ Next, use it to install/use a suitable version of **Node.js** as listed above, f
 
 *Taquito uses `lerna` internally to simplify the build configuration.*
 
-* Install `libudev-dev` if you are developing on Linux:
+* Install Nx:
+
+    `npm install -g nx`
+
+[nx](https://nx.dev/) is used to speed up the build, test and overall DX.
+
+* Install required plugins:
```

Comment:
I wonder if this is necessary for the README as the dependencies are specified in the package.json they will be installed with npm ci.

Reply:
That's a good point and this should be removed.
I will do that early next week (when I am better connected).

Reply:
This has been elided from the README.md (n.b. this file is about to change drastically anyway ^_^).